pyodide_build/config.py

```diff
@@ -204,10 +204,10 @@ def to_env(self) -> dict[str, str]:
     "cmake_toolchain_file": str(TOOLS_DIR / "cmake/Modules/Platform/Emscripten.cmake"),
     "meson_cross_file": str(TOOLS_DIR / "emscripten.meson.cross"),
     # Rust-specific configuration
-    "rustflags": "-C link-arg=-sSIDE_MODULE=2 -C link-arg=-sWASM_BIGINT -Z link-native-libraries=no",
+    "rustflags": "-C link-arg=-sSIDE_MODULE=2",
```

Comment:
Actually, I guess this will make building for existing versions annoying. I'll leave these defaults here and just set `RUSTFLAGS` in `Makefile.envs`.